#! /bin/sh /usr/share/dpatch/dpatch-run
## no_rpath.dpatch by Stefano Zacchiroli <zack@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Don't use rpath.

@DPATCH@
diff -urNad trunk~/tools/ocamlmklib.mlp trunk/tools/ocamlmklib.mlp
--- trunk~/tools/ocamlmklib.mlp	2007-02-07 11:31:36.000000000 +0100
+++ trunk/tools/ocamlmklib.mlp	2007-04-11 11:10:26.000000000 +0200
@@ -33,6 +33,11 @@
 and implib = ref ""         (* windows implib flag *)
 and verbose = ref false
 
+(* Debian specific: inhibit rpath *)
+let byteccrpath = ""
+and nativeccrpath = ""
+and mksharedlibrpath = ""
+
 let starts_with s pref =
   String.length s >= String.length pref &&
   String.sub s 0 (String.length pref) = pref
